recipes-wpe/wpeframework/wpeframework-plugins_git.bbappend

```diff
@@ -1,15 +1,13 @@
-
-WPE_COMPOSITOR = "${@bb.utils.contains('DISTRO_FEATURES', 'wayland', 'compositor_wayland', 'compositor_rpi', d)}"
-
 PACKAGECONFIG_append = " ${@bb.utils.contains('DISTRO_FEATURES', 'compositor', '${WPE_COMPOSITOR}', '', d)} \
-                        displayinfo playerinfo"
-
+                        dialserver dial_youtube displayinfo playerinfo power"
```

Comment:
Could you please move static part of PACKAGECONFIG to the .conf file?

Reply:
done